src/state-incentive-details.tsx

```diff
@@ -506,8 +414,10 @@ const IncentiveGrid: FC<IncentiveGridProps> = ({
   emailSubmitter,
 }) => {
   return tabs.length > 0 ? (
-    <div className="grid-section" id={htmlId}>
-      <h2 className="grid-section__header">{heading}</h2>
+    <div className="min-w-52" id={htmlId}>
```

Comment:
This is 208px, although the original min-width of this was 200px. I think that's fine.